### PR TITLE
Fixing generatebundle ECR tests to work in prow/mocks

### PR DESCRIPTION
**Pull request by jonahjon**
Signed-off-by: jonahjon <jonahjones094@gmail.com>

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by a-cool-train** [comment]
@jonahjon can you a description of the tests being added? It will help with reviewing.

**Comment by jonahjon** [comment]
@a-cool-train Same tests as before, it's just adding Mocks to them instead of using real SDK calls.

File: generatebundlefile/ecr_test.go [hunk 1]
```diff
@@ -239,3 +230,129 @@ func TestTagFromSha(t *testing.T) {
 		})
 	}
 }
+
```
[the remark this answers is not in the document]
Comment (TerryHowe) [reply generatebundlefile/ecr_test.go:233]: You just need to have an interface to use mockgen

[the remark this answers is not in the document]
Comment (jonahjon) [reply generatebundlefile/ecr_test.go:233]: ah yea I was using a different mock gen type tool recommend by Eric for this.

**Review by TerryHowe — APPROVED** [review]
/lgtm
/approve

**Pull request merged**